sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/models/ImagePayloadListDataModel.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.azure.ai.openai.implementation.models;
+
+import com.azure.core.annotation.Immutable;
+import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.List;
+
+/** The ImagePayloadListDataModel model. */
+@Immutable
+public final class ImagePayloadListDataModel extends DataModelBase {
```

Comment:
Do you know why we don't include `ImagePayload` when we introduce the image generation API? I saw it added since the 2023-06-01-preview service version. @jpalvarezl